BUILD.md

```diff
@@ -40,38 +41,72 @@ The `npm run watch` script in action:
-    * [Download nvm using homebrew](https://sukiphan.medium.com/how-to-install-nvm-node-version-manager-on-macos-d9fe432cc7db)
-    * Using nvm install and use specific version of node: 
-      ```
-      nvm install 18.19.0
-      nvm use 18.19.0
-      ```
-  * Yarn
-    * `npm run enable-corepack` (from within the project directory)
-  * [Docker Engine is available on Mac OS through Docker Engine](https://docs.docker.com/desktop/install/mac-install/). 
-    * _See the difference in system requirements for Docker Desktop for Intel and ARM Macs under System Requirements on the page above._
-  * Docker Compose
-    * Installing Docker Desktop on Mac will include Docker Compose 
-  * OpenJDK (Corda support Java 8 JDK but do not currently support Java 9 or higher)
-    * [Follow instructions for Mac here](https://github.com/supertokens/supertokens-core/wiki/Installing-OpenJDK-for-Mac-and-Linux)
-  * Indy SDK (optional)
-    * [Build the SDK from source](https://github.com/hyperledger-archives/indy-sdk/blob/main/docs/build-guides/mac-build.md)
-  * Go
-    * [Installing Go for Mac](https://go.dev/dl/)
-      * _Under featured downloads on the page above choose between the ARM64 or x86-64 option based on your machine._
-    * [Adding Environment Variable and Go extensions](https://code.visualstudio.com/docs/languages/go)
+* Git
+  * https://github.com/git-guides/install-git#install-git-on-mac
+* NodeJS v18.19.0, npm v10.2.3 (we recommend using the Node Version Manager (nvm) if available for your OS)
+  * [Download nvm using script](https://github.com/nvm-sh/nvm?tab=readme-ov-file#install--update-script)
+    * _See the Section "Macs with Apple Silicon Chips" under [macOS Troubleshooting](https://github.com/nvm-sh/nvm?tab=readme-ov-file#macos-troubleshooting) for ARM Mac specific instructions_
+  * [Download nvm using homebrew](https://sukiphan.medium.com/how-to-install-nvm-node-version-manager-on-macos-d9fe432cc7db)
+  * Using nvm install and use specific version of node: 
+    ```
+    nvm install 18.19.0
+    nvm use 18.19.0
+    ```
+* Yarn
+  * `npm run enable-corepack` (from within the project directory)
+* [Docker Engine is available on Mac OS through Docker Engine](https://docs.docker.com/desktop/install/mac-install/). 
+  * _See the difference in system requirements for Docker Desktop for Intel and ARM Macs under System Requirements on the page above._
+* Docker Compose
+  * Installing Docker Desktop on Mac will include Docker Compose 
+* OpenJDK (Corda support Java 8 JDK but do not currently support Java 9 or higher)
+  * [Follow instructions for Mac here](https://github.com/supertokens/supertokens-core/wiki/Installing-OpenJDK-for-Mac-and-Linux)
+* Indy SDK (optional)
+  * [Build the SDK from source](https://github.com/hyperledger-archives/indy-sdk/blob/main/docs/build-guides/mac-build.md)
```

Comment:
Indy SDK has been replaced with Aries some time ago, we can remove this requirement